test/views/ErrorTemplateViewSpec.scala

```diff
@@ -65,7 +65,7 @@ class ErrorTemplateViewSpec extends AnyWordSpec with Matchers with GuiceOneAppPe
     lazy implicit val document: Document = Jsoup.parse(view.body)
 
     "have the correct page title" in {
-      elementText(Selectors.pageTitle) shouldBe "Update and Submit an Income Tax Return"
+      elementText(Selectors.pageTitle) shouldBe "Update and submit an Income Tax Return"
```

Comment:
Should this be updated on line 46 as well?

Reply:
Correct